### ISSUE TITLE
Stability : Tracing

### ISSUE THREAD

**Issue by hdost** [open-telemetry/opentelemetry-rust#977]
So I am a but concerned that a formatting change may have "upgraded" the status of Tracing 

https://github.com/open-telemetry/opentelemetry.io/commit/7deb7952c068c59a34293a9d761f97942ef70274

Seems like a copy paste issue. 
But to my knowledge rust has not gone stable with Tracing yet.

**Comment by hdost** [comment]
@open-telemetry/rust-approvers It's been a couple months like this, I know I'm not  as active as the rest of you, but I feel like this wasn't intended.

**Comment by jtescher** [comment]
Yeah we're not currenty 1.0, but we're pretty close to stable with tracing could probably commit to a 1.0 at this point and do a 2.0 if we need breaking changes.

**Comment by hdost** [comment]
Perhaps we could add any last tickets to get it to Stable in https://github.com/open-telemetry/opentelemetry-rust/milestone/1

**Comment by TommyCpp** [comment]
Yeah I like the idea to be more organzied on our milestone.

**Comment by hdost** [comment]
I added basically everything related to core SDK functionality to the stabilization, if there's anything that could done after then I'd say feel free to add labels.

**Comment by hdost** [comment]
For spec adherence maybe we stick with a specific release for now https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.26.0 is the lastest. And then if something comes in then we can can defer for the purpose of getting to stable. Typically this will be better since the spec is only point releases.
